index.html

```diff
@@ -26,16 +26,131 @@ <h1 class="primaryHeading">
+
+            <div class="divList">
+                <p class="canopy">canopy</p>
+                <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+                <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+            </div>
+
+            <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="language">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javacript</li>
+            </ul>
+            <a href="#" class="seeProject">See Project</a>
+        </div>
+
+
+    <div class="card">
+        <div class="image3 img"></div>
+        <h1 class="tonic">Multi-Post Stories</h1>
+
+        <div class="divList">
+            <p class="canopy">canopy</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+        </div>
+
+            <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="language">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javascript">javacript</li>
+        </ul>
+        <a href="#" class="seeProject">See Project</a>
+    </div>
+
+
+    <div class="card">
+        <div class="image4 img"></div>
+        <h1 class="tonic">Tonic</h1>
+
+        <div class="divList">
+            <p class="canopy">canopy</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+        </div>
+
+        <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="language">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javascript">javacript</li>
+        </ul>
+        <a href="#" class="seeProject">See Project</a>
+    </div>
+
+
+    <div class="card">
+        <div class="image5 img"></div>
+        <h1 class="tonic">Multi-Post Stories</h1>
+
+        <div class="divList">
+            <p class="canopy">canopy</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+        </div>
+
+        <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="language">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javascript">javacript</li>
+        </ul>
+        <a href="#" class="seeProject">See Project</a>
+    </div>
+</section>  
+
+<section class="about">
+    <h1 class="aboutHeading">
```

Comment:
- [ ] Please, it would be nice and advisable to take into consideration that the h1 tag should only be used once on a page. This is because h1 is for the main title and other h can take from h2 to h6. In that sense, I think it would be nice to change this h1.